verification/verify_gauge.py

```
@@ -0,0 +1,78 @@
+import time
+import subprocess
+import os
+import signal
+from playwright.sync_api import sync_playwright
+
+def verify_gauge():
+    # Start the mock app
+    # Using preexec_fn=os.setsid to create a new process group so we can kill the whole tree
+    process = subprocess.Popen(
+        ["python", "mock_app.py"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)